src/java.base/share/classes/java/security/spec/PSSParameterSpec.java

```diff
@@ -221,10 +221,10 @@ public int getTrailerField() {
     @Override
     public String toString() {
         StringBuilder sb = new StringBuilder();
-        sb.append("MD: " + mdName + "\n")
-                .append("MGF: " + mgfSpec + "\n")
-                .append("SaltLength: " + saltLen + "\n")
-                .append("TrailerField: " + trailerField + "\n");
+        sb.append("MD: " + mdName + ", ")
```

Comment:
I would replace "MD" with "HashAlgorithm" to be consistent with the names of the other fields.